pushmi_pullyu.gemspec

```diff
@@ -24,9 +24,13 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.3.1'
 
   spec.add_runtime_dependency 'activesupport', '~> 5.0'
+  spec.add_runtime_dependency 'connection_pool', '~> 2.2'
   spec.add_runtime_dependency 'daemons', '~> 1.2', '>= 1.2.4'
   spec.add_runtime_dependency 'redis', '~> 3.3'
-  spec.add_runtime_dependency 'connection_pool', '~> 2.2'
+  spec.add_runtime_dependency 'rollbar', '~> 2.14'
+
+  # Rollbar recommended json serializer & version
+  spec.add_runtime_dependency 'oj', '~> 2.12.14'
```

Comment:
They're pinned to that because of a bug in later versions that breaks AR serialization, 2.13 introduced, then fixed, then 2.14 introduced some others, etc, etc.

Unfortunately it looks like we need to drop Ruby 2.4 or go with slower Json generation/error reporting